### ISSUE TITLE
[Question] - Is it possible to cache inferences for TTS

### ISSUE THREAD

**Issue by arnab-winimy** [triton-inference-server/server#4014]
Currently we are trying to use Triton to infer Phrases to produce audio files. Is it possible to cache the response/result of same phrase at the Triton server level? If not are there any recommended way to do this type of caching.

**Comment by Tabrizian** [comment]
Yes, it is possible to cache inferences. There is a documentation about it in the link below:
https://github.com/triton-inference-server/server/blob/main/docs/response_cache.md

cc @rmccorm4
